webapp/channels/src/sass/base/_structure.scss

```diff
@@ -160,7 +160,6 @@ body.app__body #root {
             }
             #SidebarContainer.move--right {
                 position: relative;
-                left: 65px;
```

Comment:
Will this affect anything else that may be affected by this class?

Reply:
@marianunez I don't think so, since the 'move--right' style is directly attached to the #SidebarContainer id and the only occurence of this id is specifically to the SidebarContainer https://github.com/mattermost/mattermost-server/blob/master/webapp/channels/src/components/sidebar/sidebar.tsx#L239